app/views/hub/users/edit.html.erb

```diff
@@ -16,15 +16,22 @@
         <% end %>
       <% end %>
 
-      <%= form_with model: [:hub, @user], method: :put, local: true, builder: VitaMinFormBuilder, html: { class: 'form-card form-card--long' } do |f| %>
-        <p>
-          <strong><%= t("general.email") %>:</strong> <%= @user.email %>
-          <%= link_to t("general.resend_invitation"), user_profile_resend_invitation_path(user_id: @user.id), class: "button button-small", method: :put %>
-        </p>
-
-        <%= f.cfa_input_field(:name, t("general.name")) %>
-        <%= f.cfa_input_field(:phone_number, t("general.phone_number"), type: "phone") %>
-        <%= f.cfa_select(:timezone, t("general.timezone"), timezone_select_options) %>
+      <%= form_with model: [:hub, @user], method: :put, local: true, builder: VitaMinFormBuilder, html: { class: 'form-card' } do |f| %>
+        <div>
```

Comment:
This replaces the email (that cannot be edited) with a disabled input with greyed/not-allowed cursor AND only displays the resend invite button if the client has not already accepted the invite.